develocity-gradle.yml

```diff
@@ -349,12 +493,15 @@ spec:
   }
 
   function inject_develocity_for_gradle() {
+    export "DEVELOCITY_INJECTION_ENABLED="true"
+    export "DEVELOCITY_INJECTION_INIT_SCRIPT_NAME=init-script.gradle"
+    export "DEVELOCITY_AUTO_INJECTION_CUSTOM_VALUE=GitLab"
     export "DEVELOCITY_URL=$[[ inputs.url ]]"
     export "DEVELOCITY_PLUGIN_VERSION=$[[ inputs.gradlePluginVersion ]]"
     export "DEVELOCITY_CCUD_PLUGIN_VERSION=$[[ inputs.ccudPluginVersion ]]"
     export "DEVELOCITY_ALLOW_UNTRUSTED_SERVER=$[[ inputs.allowUntrustedServer ]]"
     export "DEVELOCITY_ENFORCE_URL=$[[ inputs.enforceUrl ]]"
-    export "DEVELOCITY_CAPTURE_TASK_INPUT_FILES=$[[ inputs.captureTaskInputFiles ]]"
+    export "DEVELOCITY_CAPTURE_FILE_FINGERPRINTS=$[[ inputs.captureTaskInputFiles ]]"
```

Comment:
I would also rename the GitLab template input to `captureFileFingerprints`

Reply:
Changed in https://github.com/gradle/develocity-gitlab-templates/pull/25/commits/2471ed4d083bc937062bcf0203ad96d9ed7156b8